app/wrappers/wakes/google_analytics_api_wrapper.rb

```diff
@@ -19,8 +20,41 @@ def get_pageviews_for_path(path, start_date:, end_date:, profile_id: Wakes.confi
     ).totals_for_all_results['ga:pageviews'].to_i
   end
 
+  def get_page_of_pageviews(page, start_date:, end_date:, profile_id: Wakes.configuration.ga_profiles['default'])
+    results = authorized_analytics_service.get_ga_data(
+      "ga:#{profile_id}",
+      format_date(start_date),
+      format_date(end_date),
+      'ga:pageviews',
+      :dimensions => 'ga:pagePath',
+      :sort => '-ga:pageviews',
+      :start_index => start_index_for_page(page)
+    )
+    create_page(results.rows || [])
+  end
+
   private
 
+  def create_page(rows)
+    Page.new(page_views_from_rows(rows), rows.size < PAGE_SIZE)
```

Comment:
@joeosburn-desiringgod I'm surprised they don't give you way to know if you're on the last page. Are you sure they don't have page number and total page count fields?